apps/cards.py

```diff
@@ -956,15 +908,21 @@ def card_id1(object_data, object_uppervalid, object_upper):
             html.Div(badges),
             dcc.Markdown(
                 """
-                ```python
-                Discovery date: {}
-                Last detection: {}
-                Number of detections: {}
-                Number of low quality alerts: {}
-                Number of upper limits: {}
-                ```
+                Discovery date: `{}`
+                Last detection: `{}`
+                Duration: `{:.2f}` / `{:.2f}` days
```

Comment:
The redesign is great! However, I could not understand what the 2 numbers in `Duration: 15.94 / 19.86 days` mean? g & r duration?

Reply:
Second one is the difference between `jdendhist` and `jdstarthist` - so includes the points potentially not present in Fink, but probably important for understanding the long-term behaviour